.travis.yml

```
@@ -1,34 +1,37 @@
-sudo: required
-dist: trusty
+os: linux
+dist: bionic
+
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+
+services:
+  - mysql
+  - postgresql
 
 language: ruby
-cache: bundler
 
-addons:
-  postgresql: 9.4
+rvm:
+  - 2.5.3
+  - 2.4.5
+  - 2.3.8
```

```suggestion
  - 2.6
  - 2.5
  - 2.4
  - 2.3
```

this way it will always use the newest versions